packages/fuel-gauge/src/utils.ts

```diff
@@ -1,12 +1,8 @@
 import type { BytesLike } from '@ethersproject/bytes';
```

Comment:
We can import `BytesLike` from `fuels` and remove depedency for `@ethersproject/bytes`